internal/clients/api_client.go

```diff
@@ -379,39 +381,41 @@ func buildKibanaClient(cfg config.Client) (*kibana.Client, error) {
 	}
 
 	if logging.IsDebugOrHigher() {
-		kib.Client.SetDebug(true)
+		// Don't use kib.Client.SetDebug() here as we re-use the http client within the OpenAPI generated clients
```

Comment:
Sorry, I don't get it - I guess what we are dong in the following 2 lines has the same result - setting debug mode for Kibana client that is shared between different generated clients (that is totally fine IMO).

Reply:
Yep, the logs will look a little different from what they are now, but ultimately we'll be logging the same things (request/response pairs).